enrollment-server/pom.xml

```diff
@@ -43,10 +43,18 @@
             <artifactId>enrollment-server-domain-model</artifactId>
             <version>${project.version}</version>
         </dependency>
+
+        <dependency>
+            <groupId>com.wultra.security</groupId>
+            <artifactId>enrollment-server-common</artifactId>
+        </dependency>
+
+        <!-- TODO (racansky, 2022-06-21) not sufficient isolation, enrollment shares some logic from onboarding but including all classes -->
         <dependency>
             <groupId>com.wultra.security</groupId>
-            <artifactId>enrollment-server-api-model</artifactId>
+            <artifactId>enrollment-server-onboarding</artifactId>
```

Comment:
I am not quite happy with this solution, but might be good enough as the very fist step.

Reply:
We can improve the separation later.